### PR TITLE
Change logo

### PR DESCRIPTION
**Pull request by goldfishdolphin**
#### Before 
Screenshot 2023-04-12 at 3 23 59 pm

#### After

Screenshot 2023-04-12 at 3 45 03 pm

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
Thank you, @goldfishdolphin. An enhancement to consider in support of light and dark mode browsers --> https://github.com/meshery/meshery/blob/4c78fbcd5ed8e16eae7a875f557d36da50b2328b/README.md?plain=1#L41-L42

**Comment by goldfishdolphin** [comment]
> Thank you, @goldfishdolphin. An enhancement to consider in support of light and dark mode browsers --> https://github.com/meshery/meshery/blob/4c78fbcd5ed8e16eae7a875f557d36da50b2328b/README.md?plain=1#L41-L42

Thank you. I was also thinking about.

File: README.md [hunk 1]
```diff
@@ -16,9 +19,13 @@
 
 <p style="clear:both;">
 <h2><a href="https://layer5.io/meshery">Meshery</a></h2>
-<a href="https://meshery.io"><img src="https://raw.githubusercontent.com/meshery/meshery-cpx/master/img/readme/meshery-logo-light-text.svg"
-style="margin:10px;" width="125px" 
-alt="Meshery - the Service Mesh Management Plane" align="left" /></a>
+<a href="https://meshery.io">
+<picture><img src="./img/readme/meshery-logo-light-text.svg#gh-dark-mode-only"
+style="margin:10px;" width="10%" 
+alt="Meshery - the Service Mesh Management Plane" align="left" />
```
Comment (leecalcote) [inline README.md:25]:
```suggestion
alt="Meshery - the Cloud Native Manager" align="left" />
```

Comment (leecalcote) [reply README.md:25]: Meshery started as the service mesh manager, but has vastly expanded past service mesh to all of cloud native.

**Pull request merged**